packages/core-data/src/actions.js

```diff
@@ -441,6 +442,7 @@ export const saveEntityRecord =
 			isAutosave = false,
 			__unstableFetch = apiFetch,
 			throwOnError = false,
+			forceCreate = false,
```

Comment:
@adamziel I'd really appreciate any feedback you had on this change.

Essentially because I'm including the "key" (i.e. `slug`) in the `record` I'm passing to `saveEntityRecord` it was always defaulting to `PUT` (i.e. "update" request). However in this case we want to always force a `POST` ("create") request.

I added a flag as a workaround but I'm not sure if it's a good route to go down. 

Can you think of any alternatives? What other options could I consider?

Happy to pair.

Much appreciated 🙇‍♂️

Reply:
Should it always be `POST` for the particular entity type? If yes, then you could move that to the entity config. It could be something like...

```js
{
	kind: '...',
	name: '...',
	baseURL: '...',
	updateUsingPOST: true
}
```

However, I'm not sure if that single place where you call `saveEntityRecord` is worth creating a new API from day one. Perhaps we start by leveraging `__unstableFetch`:

```js
			return saveEntityRecord( 'postType', 'wp_navigation', record, {
				__unstableFetch: ( params ) => apiFetch({
					...params,
					method: 'POST'
				}),
			} )
```

Reply:
Ok I've used unstable fetch as the approach for now. I've clearly documented this with inline comments as it's probably a little unexpected when coming to the funciton.